elephant/current_source_density/KCSD.py

```diff
@@ -0,0 +1,1274 @@
+"""This script is used to generate Current Source Density Estimates, using the
+kCSD method Jan et.al (2012).
+
+This was written by :
+[1]Chaitanya Chintaluri, 
+[2]Michal Czerwinski, 
+Laboratory of Neuroinformatics,
+Nencki Institute of Exprimental Biology, Warsaw.
+KCSD1D[1][2], KCSD2D[1], KCSD3D[1], MoIKCSD[1]
+
+"""
+from __future__ import division
+
+import numpy as np
+from scipy import special, integrate, interpolate
+from scipy.spatial import distance
+from numpy.linalg import LinAlgError
+### Dependencies of elephant!
+# try:
+#     from skmonaco import mcmiser
+#     skmonaco_available = True
+#     import multiprocessing
+#     num_cores = multiprocessing.cpu_count()
+# except ImportError:
+#     skmonaco_available = False
+skmonaco_available = False
+
+from . import utility_functions as utils
+from . import basis_functions as basis
+
+class CSD(object):
+    """CSD - The base class for CSD methods."""
+    def __init__(self, ele_pos, pots):
+        """Initialize CSD Class.
+
+        Parameters
+        ----------
+        ele_pos : numpy array
+            positions of electrodes
+        pots : numpy array
+            potentials measured by electrodes
+
+        Returns
+        -------
+        None
+        """
+        self.validate(ele_pos, pots)
+        self.ele_pos = ele_pos
+        self.pots = pots
+        self.n_ele = self.ele_pos.shape[0]
+        self.n_time = self.pots.shape[1]
+        self.dim = self.ele_pos.shape[1]
+        self.cv_error = None
+        return
```

Comment:
No `return` statement needed in a constructor of a class. You can also delete the passage in the docstrings.